src/pages/ChangePolicyEducationalModal.tsx

```diff
@@ -0,0 +1,31 @@
+import {useFocusEffect} from '@react-navigation/native';
+import React, {useCallback, useRef} from 'react';
+import {InteractionManager} from 'react-native';
+import ChangePolicyEducationalMenu from '@components/ChangePolicyEducationalMenu';
+import blurActiveElement from '@libs/Accessibility/blurActiveElement';
+import {dismissChangePolicyModal} from '@libs/actions/Report';
+import CONST from '@src/CONST';
+
+function ChangePolicyEducationalModal() {
+    const focusTimeoutRef = useRef<NodeJS.Timeout | null>(null);
+    useFocusEffect(
+        useCallback(() => {
+            focusTimeoutRef.current = setTimeout(() => {
+                InteractionManager.runAfterInteractions(() => {
+                    blurActiveElement();
```

Comment:
What is the purpose of this change? 
I can see it's used on 3 other pages, but none of them has a comment explaining which problem this overcomes 😄 
Which element do we need to focus after the transition ends?

Reply:
@rayane-d, it was added to focus the Submit button when enter key is pressed, however, I can close the modal via enter press even without this code. Could you please double-check is this is necessary for this modal?